lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
true, the part that comes after is called content type [parameter](https://tools.ietf.org/html/rfc2045#section-5.1), the most common of which is probably `charset`.